pkg/cri/server/helpers_linux.go

```diff
@@ -156,6 +156,9 @@ func (c *criService) seccompEnabled() bool {
 
 // openLogFile opens/creates a container log file.
 func openLogFile(path string) (*os.File, error) {
+	if err := os.MkdirAll(filepath.Dir(path), 0755); err != nil {
```

Comment:
Do we want this to match what we have in k/k? https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L1306-L1310

- call os.Stat
- add directory path in err (and the returned error)

Reply:
omg, calling stat to do a mkdirall is so racy, no wonder ;) No, we do not want to match kubelet

Reply:
I just didn't do an errors.Wrap since the other call in this func does not wrap the error.

Reply:
++ thanks @crosbymichael